website/content/maintainers/templates/governance-elections.md

```diff
@@ -8,6 +8,260 @@ aliases:
+work for project members and maintainers whose time is often better spent on
+project development, rather than governance.
+
+Before deciding to implement this governance model for a CNCF project, please 
+reach out to [TAG Contributor Strategy] so that we can discuss how you might
+implement this for your CNCF project.
+
+## Requirements
+
+### What Do I Need To Know?
+
+There are quite a few things you need to know before creating a governance model
+of this complexity. Here is a partial list, but there are likely others depending
+on exactly how your project plans to operate:
+* A list of your current maintainers
+* Locations of your repositories and files
+* Details on your project's existing developer meetings
+* Details on mailing lists or other official communications channels, including both public and private channels for the Maintainers
+* Other committees that the Steering Committee may choose to delegate its authority to as-needed (security, code of conduct, etc).
+* Steering Committee meeting logistics - how often, when, where meeting notes will be published, etc.
+* Contact details for how community members can contact the Steering Committee
+* Composition - Number of seats, terms, etc.
+* How you will bootstrap the first steering committee
+* Eligibility criteria for members to run for seats on the Steering Committee
+* Election procedures for timing, election officers, eligibility of voters, etc.
+* Employer representation limits
+* Procedure for handling vacancies
+
+### What Do I Need to Customize?
+
+All sections of this template will likely need to be customized, so we recommend 
+that you reach out to [TAG Contributor Strategy] with questions about how to 
+customize it for your CNCF project.
+
+### What Else Is Required?
+
+This template assumes that you have already adopted the [Code of Conduct] and
+assumes that you are using the [Contributor Ladder], since it is a good way to provide
+guidance about eligibility requirements. It is possible to implement this without a
+[Contributor Ladder], but it is not recommended.
+
+## Glossary
+
+**Project**: Your entire CNCF project, rather than individual repositories or subprojects.
+
+**Steering Committee**: The overall governing body for your project.
+
+## Template Details
+
+### Introduction
+
+Your intro paragraph needs to include a succinct summary of the Steering
+Committee and the scope of its oversight / decision-making.
+
+### Values
+
+Like the other templates, you need to place a list of values here that define
+what your project strives for.  Some of these will be general (like "fairness")
+and some will be specific to your problem domain (like "asyncronous operation").
+The Values are listed in your governance template because all project leaders
+are expected to follow these values.  Deciding your values is a good topic
+for a general community meeting.
+
+See our documentation on [Charters] for some examples.
```

Comment:
This link isn't showing up as a link for me?

Reply:
Oops, there was a typo - this should be fixed now.